packages/salesforcedx-vscode-lwc/resources/static/js-meta.xsd

```diff
@@ -221,12 +230,119 @@
         <xs:documentation>Specifies whether the attribute is inputOnly or outputOnly. If you don’t specify the role attribute, the default value allows input and output. For example, if a property is restricted to outputOnly, users can’t set its value from a Lightning record page. Supported only if the target is lightning__FlowScreen. If you don’t set the role attribute, or if you set it to inputOnly, the property is exposed in a custom property editor. If you set it to outputOnly, the property isn’t exposed in a custom property editor.</xs:documentation>
       </xs:annotation>
     </xs:attribute>
-    <xs:attribute type="xs:string" name="type" use="required">
+    <xs:attribute type="propertyTypeAttr" name="type" use="required">
       <xs:annotation>
         <xs:documentation>The attribute’s data type. Please refer to the documentation for all supported types for different targets.</xs:documentation>
       </xs:annotation>
     </xs:attribute>
   </xs:complexType>
+  <!-- type attribute on property XSD -->
+  <xs:simpleType name="propertyTypeAttr">
+    <xs:union>
+      <xs:simpleType>
+        <xs:restriction base="xs:string">
+          <xs:enumeration value="Boolean">
+            <xs:annotation>
+              <xs:documentation>A boolean value.</xs:documentation>
```

Comment:
You need to have a documentation annotation here, or the hover text on the attribute value shows the default annotation of the `filter` attribute itself which looks funny.